modules/Campaigns/views/view.newsletterlist.php

```diff
@@ -42,31 +38,36 @@
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 
-/**
- */
+if (!defined('sugarEntry') || !sugarEntry) {
+    die('Not A Valid Entry Point');
+}
 
+require_once __DIR__ . '/../../../include/MVC/View/views/view.list.php';
 
+/**
+ * Class ViewNewsLetterList
+ */
 class ViewNewsLetterList extends ViewList
 {
     public function processSearchForm()
     {
-        // we have a query
-        if (!empty($_SERVER['HTTP_REFERER']) && preg_match('/action=EditView/', $_SERVER['HTTP_REFERER'])) { // from EditView cancel
+        if (!empty($_SERVER['HTTP_REFERER']) && preg_match('/action=EditView/',
+                $_SERVER['HTTP_REFERER'])) {
```

Comment:
Hi @Dillon-Brown , could we keep the if condition on just one line? does it go over the limit?

Reply:
Not past the limit, pushed!